specification/apicenter/resource-manager/Microsoft.ApiCenter/stable/2024-03-01/apicenter.json

```diff
@@ -3765,6 +3765,36 @@
         "lifecycleStage"
       ]
     },
+    "ApimSource": {
```

Comment:
>ApimSource [](http://example.com/codeflow?start=5&length=10)

this is not used in this swagger of this API version. is this still needed ?

Reply:
Good catch! Removed.